ship.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  publishCommand: ({ defaultCommand, tag }) =>
+    `${defaultCommand} --access public --tag ${tag}`,
+  shouldPrepare: ({ releaseType, commitNumbersPerType }) => {
+    const { fix = 0 } = commitNumbersPerType
+    if (releaseType === 'patch' && fix === 0) {
+      return false
+    }
+    return true
+  }
```

Comment:
If it's going to be a patch release, but doesn't include "fix" commits, then it will skip the release. Think about having only "chore" commits. You probably don't want to release it.

This is especially useful when you have scheduled releases.